src/server.js

```diff
@@ -408,18 +397,25 @@ function ldapAuth(client, user, password, callback) {
 	});
 }
 
-function auth(data) {
+function performAuthentication(data) {
 	const socket = this;
 	let client;
 
+	const finalInit = () => initializeClient(socket, client, !!data.remember, data.token || null);
```

Comment:
I am now passing `token` so that we have a session-like reference to it. This will be required for push notifs and other stuff later.